docs/rfcs/XeGPU.md

```diff
@@ -171,90 +142,79 @@ When the input matrix is a lower-precision data type (lower than 32bit), the inp
-  XeGPU.prefetch_nd %tdesc2 {mode = vc} : tensor_desc<16xbf16>
+  XeGPU.prefetch_nd %tdesc2: tensor_desc<8x16xbf16>
+  XeGPU.prefetch_nd %tdesc2: tensor_desc<16xbf16>
 ```
 `update_nd_offset` updates the subtensor’s offsets for the tensor descriptor. These offsets are relative offset to the current position in the number of elements.  The operation is used when the processing over one subtensor is completed and moves to a new position. Usually, only one offset value is changed since the subtensor is only moving along one dimension.
 ```mlir
-  #sg_map_a = xegpu.sg_map<wi_layout = [2, 8], wi_data = [1, 2]>
   %tdesc_updated = XeGpu.update_nd_offset %tdesc, %offsets:2 :
-  	  tensor_desc<8x16xbf16, #sg_map_a>, index, index into tensor_desc<8x16xbf16, #sg_map_a>
-
-  %tdesc_updated = XeGpu.update_nd_offset %tdesc, %offsets:2 {mode = vc} :
   	  tensor_desc<8x16xbf16>, index, index into tensor_desc<8x16xbf16>
 ```
-`create_tdesc` creates a tensor descriptor for a scattered load. It accepts a memory address and a vector of offsets. The element data type and size are specified in the output tensor_desc data type, and they must be known at the compile-time. `create_tdesc` is VC mode only.
+`create_tdesc` creates a tensor descriptor for a scattered load. It accepts a memory address and a vector of offsets. The element data type and size are specified in the output tensor_desc data type, and they must be known at the compile-time. 
 ```mlir
-  %scatter_tdesc0 = XeGPU.create_tdesc %mem_addr, %offsets, mode = vc} :
+  %scatter_tdesc0 = XeGPU.create_tdesc %mem_addr, %offsets} :
      	uint64, Vector<16 x index>, into tensor_desc<16 x uint8, #scattered>
 ```
 The example above creates a tensor_desc, which describes the memory base address and offsets for 16 uint8 values in the memory.  The number of work items (SIMD lanes) can be 1, 2, 4, 8, 16, 32.
 ```mlir
 
   #tdesc_attr = !xegpu.tdesc_attr< memory_scope=slm, scattered=true>
   %scatter_tdesc_chunk = XeGPU.create_tdesc, %base_addr, %offsets
-		{chunk_size_per_lane=8, mode = vc} :
+		{chunk_size_per_lane=8} :
 		uint64, vector<16xindex> into tensor_desc<16x8xuint16, #tdesc_attr>
 ```
 Attribute `memory_scope` indicates whether the tensor is located in the global (default) or shared local memory.
 
 Attribute `chunk_size_per_lane` specifies the size being loaded per each work item (WI). Its default value is 1, but can be set to 2, 3, 4, 8. Each WI thread may load a consecutive chunk of data elements from the memory but put them along the column dimension.
 
-`load_gather` (aka. load) load data per each work item. The output vector size is consistent with the number of WI threads, as the output describes the data being loaded at the subgroup level. `load_gather` is VC mode only.
+`load_gather` (aka. load) load data per each work item. The output vector size is consistent with the number of WI threads, as the output describes the data being loaded at the subgroup level. 
```

Comment:
this mentioning of WI threads should be also removed I guess

Reply:
Not sure I understand your comments. I think it is fine to keep the "number of WI threads" here. These operation are still subgroup level operations unless the tensor_desc is associated with a "sg_map" attribute .